packages/NodeNameResolver/NodeNameResolver/ClassConstFetchNameResolver.php

```diff
@@ -30,7 +31,7 @@ public function resolve(Node $node, ?Scope $scope): ?string
             return null;
         }
 
-        if ($node->name instanceof Error) {
+        if (! $node->name instanceof Identifier) {
```

Comment:
@TomasVotruba this is to compatibility with `nikic/php-parser` 4.17 

https://github.com/nikic/PHP-Parser/commit/cfc54e30a4f5e5af5f9d9ce86697cfcc5f7e7c24#diff-90426ec355422e33d8adf57034dc2fe3a8be657ebcbc96f6eddcba0e3bcac72d